### ISSUE TITLE
Bug in trainer.build_model()?

### ISSUE THREAD

**Issue by anastasiadk** [carpedm20/BEGAN-tensorflow#39]
Hi,

thanks for sharing the code. I have a question about the discriminator loss for real images - `self.d_loss_real` in `trainer.build_model()`. You compute it as
`self.d_loss_real = tf.reduce_mean(tf.abs(AE_x - x))`
Before that you compute `x` to be normalized versions of real images by using `x = norm_img(self.x)`, but `AE_x` is a de-normalized version of the discriminator output for `x`. Is it a bug, or am I missing something? I'm talking about [this](https://github.com/carpedm20/BEGAN-tensorflow/commit/458eb68997b95c8820a2ea874ef3e82f79812192) commit.

Thanks in advance,
Anastasia
